src/stan/io/reader.hpp

```diff
@@ -7,1520 +7,1461 @@
 
```

Comment:
did this file get updated by this PR at all?  or did it get checked in because formatting changed?